website/content/docs/concepts/domain-model/auth-methods.mdx

```diff
@@ -31,6 +31,98 @@ The password auth method has the following additional attributes:
 
 - `min_password_length` - (required) The default is 8.
 
+### LDAP Auth Method Attributes
+
+The ldap auth method has the following additional attributes:
+
+- `state` - The state of the auth method; either inactive, active-private or
+  active-public
```

Comment:
Most other list items end in a full stop
```suggestion
  active-public.
```

Reply:
I guess, okay.